src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue530.cs

```diff
@@ -1,4 +1,5 @@
-﻿using NUnit.Framework;
+﻿#if TEST_FAILS_ON_CATALYST // Getting OpenQA.Selenium.InvalidSelectorException on Catalyst
```

Comment:
On which line?

Reply:
Encountering an OpenQA.Selenium.InvalidSelectorException on line 23 of Catalyst. We also attempted to adjust the time span for the WaitForElement, still getting same issue.